src/App.js

```diff
@@ -1,25 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import Main from './components/Main.js';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default class App extends Component {
+  render() {
+    return (
+      <div>
+        <h1>React Madness</h1>
+        <Main />
+        <p>footer</p>
+      </div>
+    )
+  }
 }
```

Comment:
Really close here!

Inside the div, we can now add the components we want as children. You already have main, so add in `Header` and `Footer` components too :) 👍